docs/B-Computations/logic.md

```diff
@@ -283,7 +287,7 @@ The C language supports three iteration constructs:
 
 Three instructions control the execution of an iteration: an initialization, a test condition, a change statement.  The test condition may be simple or compound.  The flow charts for the three constructs are shown below.
 
-![](https://ict.senecacollege.ca//~ipc144/pages/images/iteration.png)
+![iteration](../../static/img/image38.png)
```

Comment:
Suggestion to provide meaningful alt text. Example: `Flow charts of the three iteration constructs`